src/Dapr.Actors/Runtime/ActorHost.cs

```diff
@@ -22,21 +22,17 @@ public sealed class ActorHost
         /// <param name="jsonSerializerOptions">The <see cref="JsonSerializerOptions"/> to use for actor state persistence and message deserialization.</param>
         /// <param name="loggerFactory">The logger factory.</param>
         /// <param name="proxyFactory">The <see cref="ActorProxyFactory" />.</param>
-        /// <param name="daprInteractor">The <see cref="IDaprInteractor" />.</param>
-        internal ActorHost(
+        public ActorHost(
```

Comment:
Users need to be able to access this for unit testing.

Reply:
Thanks! My earlier change broke this.